src/components/SNTLmap.vue

```diff
@@ -1471,44 +1471,44 @@
       <Sidebar>
         <template v-slot:sidebarTitle>
           When is peak SWE in 2021?
         </template>
         <template v-slot:sidebarMessage>
-          <p>Peak SWE has not happened yet for many places in the western U.S. This data is complete up through <span class="emph">April 1st, 2021</span>. We've generated a reproducible data pipline to pull data and reprocess it again this summer after the melt season is over.</p>
+          <p>Peak SWE has not happened yet for many places in the western U.S. This map shows <span class="emph">April 1st, 2021</span> which has traditionally been used as an indicator of peak SWE for the season.</p>
         </template>
       </Sidebar>
       <p>
         The USGS is undertaking new efforts to advance snow science through both measuring and modeling snowpack and linking these results to streamflow. As a part of the <a
           href="https://www.usgs.gov/mission-areas/water-resources/science/usgs-next-generation-water-observing-system-ngwos?qt-science_center_objects=0#qt-science_center_objects"
           target="_blank"
-        >USGS Next Generation Water Observing System (NGWOS)</a> new spatial and temporal snow and streamflow observations in unmonitored environments are being planned.  Those observations will provide valuable data which will inform modeling of water-cycle components in the Upper Colorado River Basin. New snow monitoring will include continuous snowpack and soil moisture stations, remotely sensed mapping of snow conditions, and manual snow surveys.
+        >USGS Next Generation Water Observing System (NGWOS)</a>,new spatial and temporal snow and streamflow observations are being planned for unmonitored areas. These observations will provide valuable data to inform predictive modeling of water-cycle components in the Upper Colorado River Basin. New snow monitoring will include continuous snowpack and soil moisture stations, remotely sensed mapping of snow conditions, and manual snow surveys. In addition, USGS is involved in the development and production of the Landsat snow covered area product, available through <a href="https://www.usgs.gov/core-science-systems/nli/landsat/landsat-fractional-snow-covered-area" target="_blank"> Earth Explorer.</a>
       </p>
       <p>
-        The USGS has also been contributing to snow science research through modeling snowpack and snowmelt dynamics from local to National extents. These models use measured data of snow, landscape and vegetation as well as meteorologic variables to understand how snowpack evolves from accumulation through melt, as well as how these changes influence streamflow. 
+        The USGS also contributes to snow science research through modeling snowpack and snowmelt dynamics at local to National scales. These models use data for snow, landscape, vegetation, and meteorologic variables to understand how the annual snowpack evolves from accumulation through melt, and how these changes influence streamflow.
```

Comment:
This is where IPDS reviewer DS suggested we add something like: 

> "USGS contributionS to snow science also includes the development and production of the Landsat snow covered area product, which is available through Earth Explorer. Additional information can be found here. https://www.usgs.gov/core-science-systems/nli/landsat/landsat-fractional-snow-covered-area "


See Issue #102

Reply:
I added this to the paragraph before